### PR TITLE
Add test for create_dir_as_python_package

### PR DESCRIPTION
**Pull request by takitsuse**
## Clarification:
- Add a test to see `test_create_dir_as_python_package(directory)` creates new directory in the path `directory` and new empty file `__init__.py` in the path.

### CODE REVIEW

File: tests/unit/remoteworker/test_remote_worker.py [hunk 1]
```diff
@@ -141,3 +147,20 @@ def test_return_url_per_environment(self):
...
+        self.temp_directory = join(self.BASE_TEMP_DIR, "temp_dir")
+
+    def test_create_dir_as_python_package(self):
+        create_dir_as_python_package(self.temp_directory)
+        self.assertTrue(os.path.isfile(join(self.temp_directory, "__init__.py")))
+
+        with open(join(self.temp_directory, "__init__.py")) as f:
+            self.assertEqual(f.read(), "")
+
+        shutil.rmtree(self.temp_directory)
```
Comment (Ram81) [inline tests/unit/remoteworker/test_remote_worker.py]: @takitsuse can you add a check to test whether folder is deleted or not.

Comment (takitsuse) [reply tests/unit/remoteworker/test_remote_worker.py]: File deletion is not what `create_dir_as_python_package` does, so I would say we don't have to test it...

Comment (Ram81) [reply tests/unit/remoteworker/test_remote_worker.py]: Yeah I understood that, but next time when this test runs and remove dir fails second run can give a false positive as the folder already exists

Comment (takitsuse) [reply tests/unit/remoteworker/test_remote_worker.py]: Now I see what you meant. will change that.

**Pull request merged**